web/templates/stream_settings/announce_stream_checkbox.hbs

```diff
@@ -0,0 +1,8 @@
+<label class="checkbox">
+    <input type="checkbox" name="announce" value="announce" checked />
+    <span></span>
+    {{#tr}}
+    Announce new stream in <strong>&#35;{notification_stream}</strong>
```

Comment:
Do we want to show the icon here based on privacy of stream? We are doing similar changes at other places.

Reply:
I am not sure about this as it is not mentioned in issue description. May be @alya can answer this.

Reply:
That would be ideal, yeah.